pkg/config/provider.go

```diff
@@ -219,7 +227,8 @@ func NewProvider(schema []byte, prefix string, modulePath string, metadata []byt
 		if !matches(name, p.IncludeList) {
 			continue
 		}
-
+		// resource is to be generated, so remove it from the skipped list
+		p.skippedResourceNames = p.skippedResourceNames[:len(p.skippedResourceNames)-1]
```

Comment:
While this looked nonintuitive at first place, I could understand the why we did like that.

Another approach could be to combine if statements into one and only add the resource to skip like just before _continue_.

Reply:
Done.